src/login/logind-inhibit.c

```diff
@@ -46,7 +46,7 @@ int inhibitor_new(Inhibitor **ret, Manager *m, const char* id) {
                 .fifo_fd = -1,
         };
 
-        i->state_file = path_join("/run/systemd/inhibit", id);
+        i->state_file = path_join(PATH_RUN_SYSTEMD_INHIBIT1, id);
```

Comment:
INHIBIT1???

Reply:
no, INHIBIT, input error. i find it and already fix it. thank you for your suggestion.